### PR TITLE
changed some of the common files to pointers to community

### PR DESCRIPTION
**Pull request by mikeshng**
Signed-off-by: Mike Ng <ming@redhat.com>

### CODE REVIEW

File: common_repo_templates/DCO [hunk 1]
```diff
@@ -1,37 +1 @@
...
-(c) The contribution was provided directly to me by some other
-    person who certified (a), (b) or (c) and I have not modified
-    it.
-
-(d) I understand and agree that this project and the contribution
-    are public and that a record of the contribution (including all
-    personal information I submit with it, including my sign-off) is
-    maintained indefinitely and may be redistributed consistent with
-    this project or the open source license(s) involved.
+Refer to our Community DCO at [Developer Certificate of Origin](https://github.com/open-cluster-management/community/blob/main/DCO).
```
Comment (swopebe) [inline common_repo_templates/DCO]: does this need to be a .md file?

Comment (swopebe) [reply common_repo_templates/DCO]: (To get an active link..)

**Comment by swopebe** [comment]
One comment, approving after a couple small changes.

**Review by swopebe — APPROVED** [review]

File: common_repo_templates/DCO (hunk 1 above)
Comment (mikeshng) [reply common_repo_templates/DCO]: No, unfortunately this DCO file is usually not a md file. So I left it like this. Let see what @qiujian16 comment is when he review this.

Comment (qiujian16) [reply common_repo_templates/DCO]: it does not need to be a md file.

**Review by qiujian16 — COMMENTED** [review]
/approve

please squash the  commit

File: common_repo_templates/DCO (hunk 1 above)
Comment (mikeshng) [reply common_repo_templates/DCO]: @qiujian16 can you review this again? The DCO file is no longer in this folder.

**Pull request merged**